ListContacts/index.ts

```diff
@@ -29,10 +30,11 @@ const httpTrigger: AzureFunction = async function (
       data: contacts,
     },
   });
+
+  const contactsList: Contact[] = context.bindings.contactsList;
```

Comment:
using a for loop is fine, however this would read better as a map. Our style is to use javascript functions to solve similar problems ie map, reduce etc